creator-node/prod.env

```diff
@@ -1,3 +1,5 @@
+# DO NOT MODIFY these values, use `audius-cli get-config creator-node` for accurate values
```

Comment:
could we tweak this slightly to like "THESE MAY NOT BE THE CORRECT FINAL VALUES. DO NOT READ OR MODIFY THESE VALUES..."